spec/policies/event_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe EventPolicy do
+
+  let(:reviewer) { create(:user, :reviewer) }
+  let(:organizer) { create(:organizer) }
+  let(:admin) { create(:admin) }
+  let(:event) { create(:event) }
+
+  subject { described_class }
+
+  #permissions ".scope" do
+  #  pending "add some examples to (or delete) #{__FILE__}"
+  #end
+
+  permissions :show?, :create?, :update?, :destroy? do
+    it 'denies reviewer users' do
+      expect(subject).not_to permit(reviewer, event)
+    end
+
+    it 'allows admin users' do
```

Comment:
admins have full permission on events.  We won't allow an admin to go and access all the event specific pages but through the admin event crud UI they can do whatever they want there.